lib/pbench/agent/fs.py

```diff
@@ -21,3 +15,26 @@ def safe_mkdir(directory, clean=False):
     except OSError as e:
         if e.errno != errno.EEXIST:
             raise
+
+
+def safe_delete(filename):
+    """Delete a file safely"""
+    try:
+        os.unlink(filename)
+        return 0
+    except OSError as e:
+        if e.errno != errno.ENOENT:
+            return 1
+        return 1
```

Comment:
Did you mean to return `0` if if the file didn't exist?

Reply:
fixed